kernel/prepare/pkg.yaml

```diff
@@ -5,7 +5,7 @@ dependencies:
   - image: '{{ .TOOLS_IMAGE }}'
 steps:
   - sources:
-      - url: https://cdn.kernel.org/pub/linux/kernel/v{{ regexReplaceAll ".\\d+\\.\\d+$" .linux_version "${1}" }}.x/linux-{{ .linux_version }}.tar.xz
+      - url: https://cdn.kernel.org/pub/linux/kernel/v{{ regexReplaceAll ".\\d+(\\.\\d+)?$" .linux_version "${1}" }}.x/linux-{{ .linux_version }}.tar.xz
```

Comment:
Or we could simplify the regex to (\\.\\d+)+, that would leave only the major version, should also work, right?